.github/workflows/pipeline.yml

```diff
@@ -0,0 +1,94 @@
+name: Pipeline deployment
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches: [master]
+    types: [opened, synchronize]
+
+env:
+  ML_IMAGE_TAG: activelearning_ml:test
+  API_IMAGE_TAG: activelearning_api:test
+  ML_ENDPOINT: train_and_predict/
+  API: api/
+  WEBAPP: webapp/
+
+jobs:
+  linting:
+    runs-on: ubuntu-18.04
```

Comment:
I didn't actually consider this point to be honest, though I agree with you that it is an important one. I guess it was because I used WSL2 with ubuntu 18.04 to set up the repository locally. I can change it to 20.04 then.